kotlin/mobilesdkrs/build.gradle

```diff
@@ -160,11 +160,11 @@ signing {
 }
 
 nmcp {
-    afterEvaluate {
-        publish("release") {
-            it.username.set(System.getenv("MAVEN_USERNAME"))
-            it.password.set(System.getenv("MAVEN_PASSWORD"))
-            it.publicationType.set("AUTOMATIC")
-        }
-    }
+    //afterEvaluate {
+        //publish("release") {
+            //it.username.set(System.getenv("MAVEN_USERNAME"))
+            //it.password.set(System.getenv("MAVEN_PASSWORD"))
+            //it.publicationType.set("AUTOMATIC")
+        //}
+    //}
```

Comment:
This change shouldn't be necessary. If you were having trouble building locally, try out the new command in the README (`./gradlew buildCargoNdkDebug`)